javascript/ql/src/semmle/javascript/dataflow/DataFlow.qll

```diff
@@ -96,6 +96,7 @@ module DataFlow {
     predicate accessesGlobal(string g) { globalVarRef(g).flowsTo(this) }
 
     /** Holds if this node may evaluate to the string `s`, possibly through local data flow. */
+    pragma[nomagic]
     predicate mayHaveStringValue(string s) {
```

Comment:
my gut feeling is that we should run lgtm-full.qls/default.yml for this change. Or am I just being paranoid?

Reply:
Lets do it! https://github.com/dsp-testing/erik-krogh-dca/tree/run/inliner-esbens-suggested-run/reports  

There will definitely be some 6 hour timeouts in that run.

Reply:
wow. I am quite happy to be so wrong!